packages/suite/src/components/suite/notifications/NotificationRenderer/NotificationRenderer.tsx

```diff
@@ -119,6 +119,8 @@ export const NotificationRenderer = ({
             return success(render, notification, 'TOAST_VERIFY_MESSAGE_SUCCESS');
         case 'error':
             return error(render, notification, 'TOAST_GENERIC_ERROR');
+        case 'translated-error':
+            return error(render, notification, notification.error);
```

Comment:
The new `translated-error` goes against the convention of this `NotificationRenderer`.
`NotificationRenderer` has taxatively defined all possible notifications as single source of truth, and knows what to render accordingly, but here you are adding an escape clause – a generic configurable notification to bypass that SSoT. 

Personally, I wholeheartedly dislike the pattern of giving a component "an Id of what to render", and a very smart component that renders by Id. Ideally I'd refactor it entirely in the spirit of translated error, with more props for severity and optionally custom jsx content, and scrap all this switch case, so I love your approach!

:arrow_right:  But for the time being, let's stick to consistency please :pray: 
```suggestion
        case 'cannot-open-bluetooth-settings':
            return error(render, notification, 'TR_BLUETOOTH_CANNOT_OPEN_BLUETOOTH_SETTINGS_REMOVE_DEVICE');
```

Reply:
we are already breaking this pattern here: 
![image](https://github.com/user-attachments/assets/de4b6d7a-b3ec-46fd-9c60-efb80838d7c2)

Reply:
Yes, I did notice, but only partially. That one is still a case for specific error subject (only tor toggle), not a generic one

Reply:
ill change it